tests/testthat/test-compute_duration.R

```diff
@@ -58,3 +58,28 @@ test_that("age in weeks", {
     3
   )
 })
+
+test_that("duration in hours", {
+  expect_equal(
+    compute_duration(
+      ymd_hms("2020-12-06T9:00:00"),
+      ymd_hms("2020-12-06T13:30:00"),
+      out_unit = "hours",
+      floor_in = FALSE,
+      add_one = FALSE
+    ),
+    4.5
+  )
+})
+
+test_that("duration in days after imputation", {
+  expect_equal(
+    compute_duration(
+      convert_dtc_to_dt("2020-12", date_imputation = "first"),
+      ymd("2020-12-10"),
+      out_unit = "days"
+    ),
+    10
+  )
+})
+
```

Comment:
Can you look at the Unit Test Guidance to see how to write the tests?  ![image](https://user-images.githubusercontent.com/10111024/185801145-e4e09003-d3e7-477f-baec-af7798a565cc.png)

Please renumber the tests as appropriate.